src/nanorc/core.py

```diff
@@ -371,23 +375,27 @@ def message(self, message:str) -> NoReturn:
         """
         Append the logbook
         """
+
         if message != "":
             self.log.info(f"Adding the message:\n--------\n{message}\n--------\nto the logbook")
-            try:
-                self.logbook.add_message(
-                    message = message,
-                    apparatus = self.apparatus_id
-                )
-                # if self.runs:
-                #     if self.runs[-1].is_running():
-                #         self.runs[-1].messages.append(message)
-            except Exception as e:
-                self.log.error(f"Couldn't make an entry to elisa, do it yourself manually at {self.logbook.website}\nError text:\n{str(e)}")
+                try:
+                    self.logbook.add_message(message)
+                except Exception as e:
+                    self.log.error(f"Writing to the {self.logbook_type} logbook unsuccessful\nError text:\n{str(e)}")
 
-    def stop(self, force:bool, timeout:int, **kwargs) -> NoReturn:
+    def stop(self, force:bool, timeout:int, message="", **kwargs) -> NoReturn:
```

Comment:
Nothing should be logged on stop, the logbook has a unfortunately misleading method name, `message_on_stop` but what is referred here is the stopping the run, which happens at `drain_dataflow`